clustered/clustered-dist/src/assemble/server/conf/cluster.properties

```diff
@@ -0,0 +1,17 @@
+#Configuration for node 'covid' in stripe ID 1
+client-lease-duration=150s
+client-reconnect-window=120s
+cluster-name=cluster
+failover-priority=availability
+offheap-resources=primary-server-resource:512MB
```

Comment:
'offheap'?

Reply:
oups! saw the offheap name - forgot to put it back